portal/src/graphql/portal/ApplicationsConfigurationScreen.tsx

```diff
@@ -101,24 +98,6 @@ function makeOAuthClientListColumns(
   ];
 }
 
-function useDelayedAction(delayed: () => void): (delay: number) => void {
```

Comment:
The original version of the hook uses `useEffect` to correctly cleanup scheduled setTimeout when the component unmounts. I suggest we keep this version.